doc/reference/tooling/tt_cli/cluster.rst

```diff
@@ -1,10 +1,242 @@
-.. _tt-cluster:
+.. _tt-cluster2:
 
-Managing a cluster's configuration
-==================================
+Managing cluster configurations
+===============================
 
-..  code-block:: console
+.. code-block:: console
 
-    $ tt cluster
+    $ tt cluster COMMAND {APPLICATION[:APP_INSTANCE] | URI} [FILE] [OPTION ...]
 
-.. TODO: https://github.com/tarantool/doc/issues/3725
+``tt cluster`` manages :ref:`YAML configurations <configuration>` of Tarantool applications.
+This command works both with local configuration files in application directories
+and with centralized configuration storages (:ref:`etcd <configuration_etcd>` or Tarantool-based).
+
+``COMMAND`` is one of the following:
+
+*   ``publish``: publish a cluster configuration from a file.
+*   ``show``: print a cluster configuration.
+
+.. _tt-cluster-local:
+
+Managing local configurations
+-----------------------------
+
+``tt cluster`` can read and modify local cluster configurations stored in the
+``config.yaml`` files inside application directories.
```

Comment:
Looks like the definite article is not needed here because ```config.yaml`` files` is used in a general sense.

```suggestion
``tt cluster`` can read and modify local cluster configurations stored in
``config.yaml`` files inside application directories.
```